CedarJava/src/main/java/com/cedarpolicy/AuthorizationEngine.java

```diff
@@ -37,30 +37,30 @@
  */
 public interface AuthorizationEngine {
     /**
-     * Asks whether the given AuthorizationQuery <code>q</code> is approved by the policies and
+     * Asks whether the given AuthorizationRequest <code>q</code> is approved by the policies and
      * entity hierarchy given in the <code>slice</code>.
      *
-     * @param q The query to evaluate
+     * @param request The request to evaluate
      * @param slice The slice to evaluate against
-     * @return The result of the query evaluation
-     * @throws AuthException On failure to make the authorization query. Note that errors inside the
+     * @return The result of the request evaluation
+     * @throws AuthException On failure to make the authorization request. Note that errors inside the
      *     authorization engine are included in the <code>errors</code> field on the
-     *     AuthorizationResult. Note: This error interface will likely change in the future. We will
+     *     AuthorizationResponse. Note: This error interface will likely change in the future. We will
      *     likely unify the error handling story.
      */
-    AuthorizationResponse isAuthorized(AuthorizationRequest q, Slice slice) throws AuthException;
+    AuthorizationResponse isAuthorized(AuthorizationRequest request, Slice slice) throws AuthException;
```

Comment:
Should we rename `Slice` as well, to match the Rust API better

Reply:
Opened a new issue for this #40 (if I misinterpreted your suggestion, feel free to comment on the issue or edit it directly)

Reply:
Nope, that's exactly what I meant, looks great